src/main/resources/templates/company.html

```diff
@@ -1,51 +1,46 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
```

Comment:
XML Namespace for ThymeLeaf must be defined using `xmlns:th` not `xml:th`. [Reference](https://developer.mozilla.org/en-US/docs/Archive/Mozilla/XUL/Namespaces)